### PR TITLE
Add Fugue recap blog post

### PR DESCRIPTION
**Pull request by colebow**

### CODE REVIEW

**Review by ryandoowan — APPROVED** [review]

**Comment by ryandoowan** [comment]
Don't forget to add the "share this session" part at the end

File: _posts/2023-07-27-trino-fest-2023-fugue-recap.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+author: "Kevin Kho and Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Fugue.png
+show_pagenav: false
+---
+
+Fugue may be an unfamiliar name to those in the Trino ecosystem. It's another
+Python tool, a programming model built to enhance interoperability between
+Python and SQL. On the Python side of things, it's a wrapper around common tools
+like pandas and Polars which converts that code into SQL for high-performance,
```
Comment (mosabua) [inline _posts/2023-07-27-trino-fest-2023-fugue-recap.md:13]:
```suggestion
like pandas and Polars that converts code into SQL for high-performance,
```

File: _posts/2023-07-27-trino-fest-2023-fugue-recap.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+large-scale query execution. So why are we talking about it at Trino Fest?
+Because Fugue recently launched an integration with Trino, enabling that Python
+code to run on a high-powered Trino backend.
```
Comment (mosabua) [inline _posts/2023-07-27-trino-fest-2023-fugue-recap.md:16]: thats not what it does.. it converts into SQL and that runs on Trino.  Need to reword

File: _posts/2023-07-27-trino-fest-2023-fugue-recap.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+<!--more-->
+
+{% youtube aKhI1Phfn-o %}
+
```
Comment (mosabua) [inline _posts/2023-07-27-trino-fest-2023-fugue-recap.md:21]: Do we have slides?

**Review by mosabua — CHANGES_REQUESTED** [review]

File: _posts/2023-07-27-trino-fest-2023-fugue-recap.md (hunk 3 above)
Comment (colebow) [reply _posts/2023-07-27-trino-fest-2023-fugue-recap.md:21]: We do not

**Review by mosabua — APPROVED** [review]
Good to go. Shipping. We can follow up with slides

**Pull request merged**